components/registration-view/registration-view.jsx

```diff
@@ -9,9 +10,20 @@ function RegistrationView(){
     const [birthday, setBirthday] = useState('');
 
 
-    const handleSubmit=(e)=>{
+    const handleRegistration=(e)=>{
         e.preventDefault();
-        console.log(username, password, email, birthday);
```

Comment:
Are you able to get the birthday here? if so then you can check the parameters to make sure the names much with what this. API endpoint `https://cataflix.herokuapp.com/users` receives in the body.

simply make sure on the api level you getting the information like this

const {body: {Username, Password, Email, Birthday} } = req

this way you can the save each to the database